pkg/serviceaccount/serviceaccount_test.go

```diff
@@ -1 +1,244 @@
 package serviceaccount
+
+import (
+	"testing"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	k8sfake "k8s.io/client-go/kubernetes/fake"
+)
+
+func TestPull(t *testing.T) {
```

Comment:
Please add TestServiceAccount* to all test functions

Reply:
Done!